### ISSUE TITLE
openapi2cscontroller generates in file post Stream where I expect IFormFile 

### ISSUE THREAD

**Issue by kejto** [RicoSuter/NSwag#2239]
yaml: 
```   
  requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                file:
                  type: string
                  format: binary
```
generates c#:
```
public System.Threading.Tasks.Task<Microsoft.AspNetCore.Mvc.ActionResult<Media>> Upload([Microsoft.AspNetCore.Mvc.FromBody] System.IO.Stream body)
```

what I need is :
Upload(IFormFile body) 
how can I achieve it using openapi2cscontroller or how can I call above from postman and hit it?
so far I was trying form-data file post with no headers and getting 415

**Comment by plk** [comment]
I just added a PR to address this, coincidentally - #2241. This PR makes the binary body type configurable but default to `System.IO.Stream` as it is currently. I also need something like `ICollection<IFormFile>` and this is now possible via, e.g. `nswag.json` config file using the new `parameterBinaryBodyType` setting.

**Comment by kejto** [comment]
@plk is this funcionality inculded in latest release? I need this quite urgently

**Comment by RicoSuter** [comment]
Yes, the PR https://github.com/RicoSuter/NSwag/pull/2253 is included in v13.0.2

**Comment by kejto** [comment]
@RicoSuter  would it be possible to move this setting to OpenApiGeneratorCommandBase.cs instead?

**Comment by RicoSuter** [comment]
It's not a setting, it's just the default behavior

**Comment by kejto** [comment]
@RicoSuter I have a problem now it creates 

public System.Threading.Tasks.Task<Microsoft.AspNetCore.Mvc.ActionResult<Media>> Upload([Microsoft.AspNetCore.Mvc.FromBody]Microsoft.AspNetCore.Http.IFormFile body)

but cant hit debugger but if I I remove [Microsoft.AspNetCore.Mvc.FromBody] it works fine I think its issue caused by fix #2246
